aronnax.f90

```diff
@@ -1356,7 +1358,7 @@ subroutine evaluate_dudt(dudt, h, u, v, b, zeta, wind_x, fu, &
             + spongeTimeScale(i,j,k)*(spongeU(i,j,k)-u(i,j,k)) ! forced relaxtion in the sponge regions
         if (k .eq. 1) then ! only have wind forcing on the top layer
           ! This will need refining in the event of allowing outcropping.
-          dudt(i,j,k) = dudt(i,j,k) + wind_x(i,j)/(rho0*h(i,j,k)) ! wind forcing
+          dudt(i,j,k) = dudt(i,j,k) + 2d0*wind_x(i,j)/(rho0*(h(i,j,k) + h(i-1,j,k))) ! wind forcing
```

Comment:
@axch This is why the `pysics-test branch fails the test suite. The test suite passes using the old version of the code. The same thing has been changed in `evaluate_dvdt`, but the lack of any tests with meridional wind forcing means it doesn't get checked.

The fact that the model uses the wrong value for `h` in these calculations was discussed in #98, and apparently I implemented a fix on this branch. I'm happy that this is the correct way to do it, and will check in new blessed outputs.


Just in case GitHub doesn't put this in place, I'm referring to the line in evaluate_dudt that was
`dudt(i,j,k) = dudt(i,j,k) + wind_x(i,j)/(rho0*h(i,j,k))`
and is now
`dudt(i,j,k) = dudt(i,j,k) + 2d0*wind_x(i,j)/(rho0*(h(i,j,k) + h(i-1,j,k)))`